frontend/src/views/CreditsPage/CreditsPage.jsx

```diff
@@ -28,6 +29,7 @@ const DevelopersButtons = () => {
 const CreditsPage = () => {
   return (
     <div className="credits-page">
+      <Navbar />
```

Comment:
Avoid having such component in markup of every page - it is easier to maintain Navbar if returned in eg. App.jsx

Reply:
done & working.